src/internal_modules/roc_core/target_nouuid/roc_core/uuid.cpp

```diff
@@ -35,8 +36,9 @@ bool uuid_generare(char* buf, size_t buf_sz) {
     }
 
     uint8_t bytes[16];
-    for (size_t i = 0; i < ROC_ARRAY_SIZE(bytes); i++) {
-        bytes[i] = (uint8_t)fast_random_range(0, 255);
+    bool ok = core::secure_random(bytes, ROC_ARRAY_SIZE(bytes));
+    if (!ok) {
+        roc_panic("secure_random() failed");
```

Comment:
This error is caused by external events, so we shouldn't panic, we should return error instead.